backend/src/index.ts

```diff
@@ -86,6 +92,31 @@ app.use('/api/v1/stripe', stripeRouter);
 app.use('/api/v1/integration', integrationRouter);
 app.use('/api/v1/integration-auth', integrationAuthRouter);
 
-app.listen(PORT, () => {
-	console.log('Listening on PORT ' + PORT);
+const server = http.createServer(app);
+
+const onSignal = () => {
+  console.log('Server is starting clean-up');
+  return Promise.all([
+    // your clean logic, like closing database connections
```

Comment:
@dangtony98 You  might want to add clean-up logic here. Maybe shut down database pool, if any?

Reply:
Added DB connection clean-up